checkov/gitlab_ci/checks/job/AvoidDoublePipelines.py

```diff
@@ -17,10 +17,12 @@ def __init__(self):
 
     def scan_entity_conf(self, conf):
         c = 0
+        pipelinecheck1 = '$CI_PIPELINE_SOURCE == "merge_request_event"'
+        pipelinecheck2 = '$CI_PIPELINE_SOURCE == "push"'
         for x in conf:
             if "if" in x:
                 value = x['if']
-                if value.startswith('$CI_PIPELINE_SOURCE'):
+                if value.startswith(pipelinecheck1) or value.startswith(pipelinecheck2):
```

Comment:
```suggestion
                if value.startswith(pipeline_sources):
```
both changes and it is good to go 🚀